conventions/build.gradle.kts

```diff
@@ -1,7 +1,7 @@
 plugins {
   `kotlin-dsl`
   // When updating, update below in dependencies too
-  id("com.diffplug.spotless") version "6.9.0"
+  id("com.diffplug.spotless") version "6.11.0"
```

Comment:
waiting to see if dependabot updates the `com.diffplug.spotless:spotless-plugin-gradle` dependency below also

Reply:
I wonder if it's waiting for this PR to be merged before updating the other one?? I'm going to merge and will update manually if not.

Reply:
it seems to be updating that other dependency just fine in my test fork: https://github.com/trask/forktest/pull/4